### ISSUE TITLE
I can't find how to ignore or exclude .env files (PEBSAK?)

### ISSUE THREAD

**Issue by JamesParrott** [redhat-developer/vscode-yaml#948]
## Describe the bug
The extension's default settings incorrectly assume that all files with no extension that could possibly be YAMl, are intended to be YAML.  For example .env files are not intended to be yaml. 

I could not find a way to make it only parse files with .yaml or .yml extensions only, after looking twice through the options.  Apologies if I missesd it.  It's otherwise a great extension and caught an annoying bug for me straight away - cheers.

## Erroneous error:
[{
	"resource": ".postgres",
	"owner": "_generated_diagnostic_collection_name_#7",
	"severity": 8,
	"message": "Unexpected scalar token in YAML stream: \"POSTGRES_PORT=5432\\r\\nPOSTGRES_DB=...\\r\\nPOSTGRES_USER=...\\r\\nPOSTGRES_PASSWORD=...\"",
	"source": "YAML",
	"startLineNumber": 9,
	"startColumn": 1,
	"endLineNumber": 9,
	"endColumn": 21
}]

This is on Windows.  Switching the line endings from CRLF to LF did not fix it.

**Comment by gorkem** [comment]
I do not think it is `vscode-yaml` that is mapping the file to YAML. vscode-yaml only uses [these](https://github.com/redhat-developer/vscode-yaml/blob/b72e1b7146baa97a4f28499a01eb0d4e64a19876/package.json#L61) extensions

**Comment by JamesParrott** [comment]
Thanks Gorkem - that's really helpful.  This is exactly the setting I was unable to find.  

Unfortunately however, this problem goes away when I disable vscode-yaml, and reoccurs when I re-enable it.  So I still do suspect this is a bug.
